README.md

```diff
@@ -4,15 +4,27 @@
 The purpose of this repo is to demonstrate algorithms in C based on a fork from Kyle Louden's book Mastering Algorithms in C. 
 
 
-### Setup 
+## Run Examples
```

Comment:
agreed!  Let's also make sure our CMakeLists.txt works with cmake from the command line.